sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Release History
 
+## 1.1.0 (2023-10-10)
+
+## Features Added
+- Added `LogsIngestionAudience` for multi-cloud support to allow users to select the Azure cloud where the resource is located.
+
 ## 1.1.0-beta.1 (2023-10-10)
 
 ### Bugs Fixed
-  - Fix sovereign support for US Gov and China clouds
```

Comment:
Did we actually ship a beta with this? If so, don't we need to update the csproj now?

Reply:
updated - yes the beta was shipped